README.md

```diff
@@ -482,8 +482,8 @@ Commands:
 
 To run Cromwell in a notebook instance:
 
-* Run `terra cromwell generate-config [--dir=my/path]`
-* In a notebook instance terminal, in `cromwell.conf`, replace `{WORKSPACE_BUCKET}` with a bucket in your workspace
+* Run `terra cromwell generate-config (--workspace-bucket-name=bucket_name | --google-bucket-name=gs://my-bucket) [--dir=my/path]`
+* One of `workspace-bucket-name` or `google-bucket-name` is required. `workspace-bucket-name` is a Terra resource name. `google-bucket-name` is a Google Cloud Storage bucket. If `google-bucket-name` does not begin with the "gs://" prefix, it will be automatically added.
```

Comment:
* One of `workspace-bucket-name` or `google-bucket-name` is required to specify the bucket used by Cromwell for workflow orchestration. 

  * `workspace-bucket-name` is a Terra resource name
  * `google-bucket-name` is a Google Cloud Storage bucket. If `google-bucket-name` does not begin with the `gs://` prefix, it will be automatically added.

Reply:
Fixed.